server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/CVRToAuditList.java

```diff
@@ -160,26 +151,9 @@ public String endpointBody(final Request the_request, final Response the_respons
 
     try {
       // get the request parameters
-      final String start_param = the_request.queryParams(START);
-      final String ballot_count_param = the_request.queryParams(BALLOT_COUNT);
-      final String duplicates_param = the_request.queryParams(INCLUDE_DUPLICATES);
```

Comment:
I'm glad these are removed if they're truly not being used, I'd just like to run through the app again post-merge to see how we do.

Reply:
Agreed. I think it was fallout from `computeBallotOrder` being removed.